### DISCUSSION TITLE
Question: Flashlight availibility on F-DROID?

### DISCUSSION THREAD

**Discussion by GITMichaelK** [FossifyOrg/General-Discussion#153] [General]
Thank you for this repository and your excellent work! 
I found several applications I used former from SimpleMobileTools, but [Flashlight](https://github.com/FossifyOrg/Flashlight) is not yet available. What is the reason and (how) can this be solved?

**Comment by Aga-C** [comment]
We can't say when exactly the app will be released. You can track, what and where has been released, here: https://github.com/orgs/FossifyOrg/discussions/9#discussioncomment-8094221.

**Comment by GITMichaelK** [comment]
Thank you @Aga-C for quick response, I had seen the [page you mentioned](https://github.com/orgs/FossifyOrg/discussions/9#discussioncomment-8094221), while searching for older posts on my topic. 
Does this mean, that it needs effort from F-DROID collaborators to add the missing applications? I would like to understand workflow and responsibilities.

**Comment by Aga-C** [comment]
We haven't done the release yet. After the release, we will submit the app to F-Droid.

**Comment by GITMichaelK** [comment]
Thanks again, my confusion is because the package is [listed here](https://github.com/FossifyOrg/Flashlight) and for me looks not different than the ones I'm using already from Fossify over F-Droid. Looking for Packagename/**releases** there are differences:
* [Filemanager](https://github.com/FossifyOrg/File-Manager/releases) has releases,
* [Flashlight](https://github.com/FossifyOrg/Flashlight/releases) has none yet.

**Comment by Aga-C** [comment]
Yes, and it will be released soon as an app to download directly, alongside with Notes, Keyboard and other yet not released apps from the Fossify suite, as it was with already released apps.
